src/cpu/core_dyn_x86/risc_x86.h

```diff
@@ -994,8 +994,8 @@ static Bit8u * gen_create_jump(Bit8u * to=0) {
 	return (cache.pos-4);
 }
 
-static void gen_fill_jump(Bit8u * data,Bit8u * to=cache.pos) {
-	*(Bit32u*)data=(to-data-4);
```

Comment:
PPC: Fix needs to be aligned and native endian (not swapped to little-endian like the host_* functions do).